library/src/skins/default/src/sass/base/_defaults.scss

```diff
@@ -56,7 +56,7 @@ a{
     color: var(--sakai-text-color-1);
     text-decoration: none;
   }
-  &[href]:not(.btn, .page-link) {  // This means a[href]
+  &[href] {  // This means a[href]. Do not make this selector more specific.
```

Comment:
The not psuedo-selector has the unintended effect of making this rule highly specific! E.g., it ends up overriding other colors (look at the link colors in the All Sites popout)